CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an issue where during shutdown unregistering NetGUIDs could cause an asset load and program stall.
 - Fixed an issue where migration diagnostic tool would crash if the target actor's owner couldn't be found.
 - Fix RPC timeouts for parameters referencing assets that can be asynchronously loaded.
+- Fixed an issue for actors with bNetLoadOnClient. A dynamic subobject removed from such an actor while out of a client's view will now be properly removed on the client when the actor comes back into the client's view.
```

Comment:
Careful to move this to above the 0.13.0 into the Unreleased block when you merge up latest. (Since most of this section is now in the 0.13.0 section in latest master)